config/configgrpc/README.md

```diff
@@ -16,7 +16,7 @@ configuration. For more information, see [configtls
 README](../configtls/README.md).
 
 - [`balancer_name`](https://github.com/grpc/grpc-go/blob/master/examples/features/load_balancing/README.md)
-- `compression` (default = gzip): Compression type to use (only gzip is supported today)
+- `compression` Compression type to use among `gzip`, `lz4`, `snappy` and `zstd`
```

Comment:
`gzip` hasn't been the default compression in our codebase. By the code in [here](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configgrpc/configgrpc.go#L184-L190), we only call `grpc.UseCompressor()` when the compressionKey is in `gRPCCompressionKeyMap`, and return error if it's not.
If we set `Compression: ""`, then the code won't return an error and it will dial to grpc without Compressor setup.
I'm trying to find out in `grpc` library that whether they have a default compression method if they don't have a value for the Compressor in their dialOptions.

Reply:
I found [this](https://grpc.github.io/grpc/core/md_doc_compression.html).
> When a compression level is not specified for either the channel or the message, the default channel level none is considered: data MUST NOT be compressed.

So if we don't specify compression in `configgrpc`, then uncompressed will be the default. Should we mention in the comment? or should we not allow to send an empty string for compressor in config?